docs/detections/api/rules/rules-api-import.asciidoc

```diff
@@ -8,6 +8,8 @@ Imports rules from an `.ndjson` file. The following configuration items are also
 
 NOTE: Console supports {es} APIs only. Console doesn't allow interactions with {kib} APIs. You must use `curl` or another HTTP tool instead. For more information, refer to {kibana-ref}/console-kibana.html[Run {es} API requests]. Please also note this API supports {kibana-ref}/api.html#token-api-authentication[Token-based authentication] only.
 
+NOTE: Your role needs at least `Read` privileges for the `Action and Connectors` feature to import rules with actions. If you are importing rules without actions, no `Action and Connectors` feature privileges are required. Refer to <<enable-detections-ui>> for more information.
```

Comment:
```suggestion
NOTE: You need at least `Read` privileges for the `Action and Connectors` feature to import rules with actions. If you are importing rules without actions, no `Action and Connectors` feature privileges are required. Refer to <<enable-detections-ui>> for more information.
```

Reply:
I like this better because I imagine users don't typically think of themselves as a "role", rather they sign in with whatever creds they have and have the associated permissions. I could be wrong though. If you do end up going with my suggestion, the same thing appears below as well.

Reply:
I see your point and think both ways work. That said, I do like the more straightforward tone of your edits so I'll apply them :)